packages/web/pages/players.tsx

```diff
@@ -1,13 +1,27 @@
 import { PageContainer } from 'components/Container';
 import { PlayerList } from 'components/PlayerList';
+import { PlayerFragmentFragment } from 'graphql/autogen/types';
 import { getPlayers } from 'graphql/getPlayers';
 import { InferGetStaticPropsType } from 'next';
 import React from 'react';
 
 type Props = InferGetStaticPropsType<typeof getStaticProps>;
 
+const LIMIT = 50;
+const OFFSET = 50;
+const TOTAL_PLAYERS = 150;
+
 export const getStaticProps = async () => {
-  const players = await getPlayers();
+  const promises: Promise<PlayerFragmentFragment[]>[] = new Array(
+    TOTAL_PLAYERS / OFFSET,
```

Comment:
I don't think you need two variables here. The calculations are really based on the limit for how many to return. The offset is a variable computed value.

```suggestion
    TOTAL_PLAYERS / LIMIT,
```